lib/modules/dosomething/dosomething_organ_donation/dosomething_organ_donation.module

```diff
@@ -53,12 +59,55 @@ function _dosomething_organ_donation_build_http_client($nid) {
     'base_url' => $base_url,
     'headers' => [
       'Authorization' => $token,
+      'Content-Type' => 'application/json',
     ],
   ];
 
   return $client;
 }
 
+/**
+ * POST user's information to Organize's Registration API endpoint.
+ *
+ * @param array $values
+ *   Values from the registration form.
+ */
+function dosomething_organ_donation_post_registration($values) {
+  $client = _dosomething_organ_donation_build_http_client();
+
+  $options = [
+    'method' => 'POST',
+    'headers' => $client['headers'],
+    'data' =>
+      json_encode([
+        'email' => $values['email'],
+        'first_name' => $values['first_name'],
+        'middle_name' => $values['middle_name'],
+        'last_name' => $values['last_name'],
+        'birthdate' => $values['birthdate'],
+        'street_address' => $values['street_address'],
+        'apartment' => $values['apartment'],
+        'city' => $values['city'],
+        'state' => $values['state'],
+        'postal_code' => $values['postal_code'],
+        'license_id' => $values['license_id'],
+        'phone_number' => $dvalues['phone_number'],
+        'ssn' => $values['ssn'],
+        'place_of_birth' => $values['place_of_birth'],
+        'maiden_name_mother' => $values['maiden_name_mother'],
+        'country_of_birth' => $values['country_of_birth'],
+        'gender' => $values['gender'],
+        'source_url' => 'www.dosomething.org',
```

Comment:
this is fine for now, but I think we should confirm with organize what we should be sending here (cc @ngjo)